test/integration/update_patient_info_test.rb

```diff
@@ -138,6 +132,45 @@ class UpdatePatientInfoTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'changing fulfillment information' do
+    before do
+      @user.update role: :admin
+      @patient = create :patient, appointment_date: 2.days.from_now,
+                                              clinic_name: 'Sample Clinic 1'
+      create :pregnancy, patient: @patient,
+                         dcaf_soft_pledge: 100,
+                         pledge_sent: true
+      create :fulfillment, patient: @patient
+      visit edit_patient_path @patient
+      # find('#submit-pledge-button').click
+      # assert false
+
+      click_link 'Pledge Fulfillment'
+      check 'Pledge fulfilled'
+      fill_in 'Procedure date', with: 2.days.from_now.strftime('%m/%d/%Y')
+      select '12 weeks', from: 'Weeks along at procedure'
+      fill_in 'Abortion care $', with: '100'
+      fill_in 'Check #', with: '444-22'
+      fill_in 'Procedure date', with: 2.weeks.from_now.strftime('%m/%d/%Y')
+
+      reload_page_and_click_link 'Pledge Fulfillment'
+    end
+
+    it 'should alter the information' do
+      within :css, '#fulfillment' do
+        assert has_checked_field? 'Pledge fulfilled'
+        assert has_field? 'Procedure_date',
+                          with: 2.days.from_now.strftime('%m/%d/%Y')
+        assert_equal '12 weeks',
+                     find('#patient_fulfillment_gestation_at_procedure').value
+        assert has_field? 'Abortion care $', with: 100
+        assert has_field? 'Check #', with: '444-22'
+        assert has_field? 'Procedure date',
+                          with: 2.weeks.from_now.strftime('%m/%d/%Y')
```

Comment:
As @ashlynnpai pointed out, you check two different `Procedure Date`s in this block, one as `Procedure_date` at line 162 and then `Procedure date` at 168/9. Presumably this will check the data you update lines 150/154 to insert. Or if it's just an accidental duplication of `Procedure date`, that's an easy fix!